<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Launcher</class>
    <widget class="QWidget" name="Launcher">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>749</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string comment="Launcher">Launcher</string>
        </property>
        <property name="styleSheet">
            <string notr="true">* {
                font-size: 16px
                }
            </string>
        </property>
        <property name="locale">
            <locale language="English" country="UnitedStates"/>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="100,500">
            <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
                    <property name="sizeConstraint">
                        <enum>QLayout::SetMinimumSize</enum>
                    </property>
                    <item>
                        <layout class="QFormLayout" name="formLayout">
                            <property name="fieldGrowthPolicy">
                                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                            </property>
                            <property name="rowWrapPolicy">
                                <enum>QFormLayout::DontWrapRows</enum>
                            </property>
                            <property name="labelAlignment">
                                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                            </property>
                            <property name="horizontalSpacing">
                                <number>10</number>
                            </property>
                            <property name="verticalSpacing">
                                <number>10</number>
                            </property>
                            <property name="leftMargin">
                                <number>2</number>
                            </property>
                            <property name="topMargin">
                                <number>10</number>
                            </property>
                            <property name="rightMargin">
                                <number>10</number>
                            </property>
                            <property name="bottomMargin">
                                <number>10</number>
                            </property>
                            <item row="0" column="0">
                                <widget class="QLabel" name="IPAddressLabel">
                                    <property name="text">
                                        <string comment="ListenAddress">Listen Address</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QLineEdit" name="AddressLineEdit">
                                    <property name="inputMask">
                                        <string/>
                                    </property>
                                    <property name="text">
                                        <string notr="true"/>
                                    </property>
                                    <property name="maxLength">
                                        <number>64</number>
                                    </property>
                                    <property name="placeholderText">
                                        <string notr="true"/>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="PortLabel">
                                    <property name="text">
                                        <string comment="ListenPort">Listen Port</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QLineEdit" name="PortLineEdit">
                                    <property name="inputMask">
                                        <string/>
                                    </property>
                                    <property name="text">
                                        <string notr="true"/>
                                    </property>
                                    <property name="maxLength">
                                        <number>5</number>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="PasswordLabel">
                                    <property name="text">
                                        <string comment="Password">Password</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QLineEdit" name="PasswordLineEdit">
                                    <property name="text">
                                        <string notr="true"/>
                                    </property>
                                    <property name="maxLength">
                                        <number>16</number>
                                    </property>
                                    <property name="echoMode">
                                        <enum>QLineEdit::Password</enum>
                                    </property>
                                    <property name="placeholderText">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QPushButton" name="ServerButton">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>100</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>Start Server</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="1" column="0">
                <widget class="QTextEdit" name="LogTextEdit">
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
